src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetStatisticsRDD.scala

```diff
@@ -167,6 +167,9 @@ class ParquetStatisticsRDD(
 }
 
 private[parquet] object ParquetStatisticsRDD {
+  // Supported top-level Spark SQL data types
+  val SUPPORTED_TYPES: Set[DataType] = Set(IntegerType, LongType, StringType)
```

Comment:
Again, this should be moved to some utility class that can be reused across different implementations.